src/main/java/frc/robot/commands/TeleopSwerve.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.commands;
+
+import frc.robot.Constants;
+import frc.robot.subsystems.Swerve;
+
+import java.util.function.BooleanSupplier;
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+
+
+public class TeleopSwerve extends Command {    
```

Comment:
We're going to have to make a lot of changes to this command, but we can do that later. Our new teleop default command will have to use limelight info to do some auto-rotating.